cli/api/api.go

```diff
@@ -173,7 +173,7 @@ func (a *api) StartServer() error {
 }
 
 // Opens a connection to the master if not already connected
-func (a *api) connectMaster() (*master.Client, error) {
+func (a *api) connectMaster() (master.ClientInterface, error) {
```

Comment:
From a language perspective, `master.Client` is an implementation of an RPC client.  By returning an interface here, I can inject a mock for unit testing methods in the api package which make calls to the RPC client.    For instance, see TestGetEndpoints_fails in service_test.go